appctl

```diff
@@ -70,6 +70,7 @@ intro() {
     echo "    ${BOLD}restart${NORMAL}           stop and start all docker containers."
     echo "    ${BOLD}rebuild${NORMAL}           rebuild and restart Misago container."
     echo "    ${BOLD}stats${NORMAL}             see list and stats of running docker containers."
+    echo "    ${BOLD}flushredis${NORMAL}        flush the redis cache."
```

Comment:
I am not sure `flushredis` should live under "Docker management" category ;)

Adding it at the end of "Shortcuts" may be better here. We could rename it to "Utilities" so it fits better.

Reply:
Good call :)